README.md

```diff
@@ -1 +1,23 @@
 # hpqtypes-effectful
+
+## Description
+
+`effectful` bindings for the `hpqtypes` haskell bindings for `libpqtypes`
```

Comment:
What about: "This library provides an `EffectDB` effect that allows to use the `hpqtypes` bindings for `libpqtypes` in the [`effectful`](https://github.com/haskell-effectful/effectful) ecosystem."?

Btw., maybe we should replace `EffectDB` with just `DB` (not necessarily in this PR)?

Reply:
Yes, revised  it to mostly your suggestion. The latter suggestion is now #20